src/Microsoft.ML.Transforms/Text/doc.xml

```diff
@@ -184,5 +184,37 @@
       </example>
     </example>
 
+    <member name="WordEmbeddings">
+      <summary>
+        Word Embeddings transform is a text featurizer which converts vectors of text tokens into sentence vectors using a pre-trained model.
+      </summary>
+      <remarks>
+        WordEmbeddings wrap different embedding models, such as GloVe. Users can specify which embedding to use. 
+        The available options are various versions of <a href="https://nlp.stanford.edu/projects/glove/">GloVe Models</a>, <a href="https://en.wikipedia.org/wiki/FastText">FastText</a>, and <a href="http://anthology.aclweb.org/P/P14/P14-1146.pdf">Sswe</a>.
+        <para>
+          Note: As WordEmbedding requires a column with text vector, e.g. %3C'This', 'is', 'good'%3E, users need to create an input column by:
```

Comment:
>' [](start = 85, length = 1)

apostrophes need be encoded too: &apos;

Reply:
hope %27 will work

---
In reply to: [204168630](https://github.com/dotnet/machinelearning/pull/545#discussion_r204168630) [](ancestors = 204168630)